### ISSUE TITLE
Late Breaking Demo

### ISSUE THREAD

**Issue by stefan-balke** [ismir/conference-archive#27]
We are currently not archiving LBDs.
Maybe this can stay a "manual process" and will not involve DBLP but maybe we want those on Zenodo?

**Comment by ejhumphrey** [comment]
worth re-raising to the board, but I believe the general sentiment is that LBD papers should be deemphasized since they are not peer reviewed. If only for practicality though, I'm happy to stick with "peer reviewed papers are the only ones to get DOIs" until there's sufficient clamoring for more.

**Comment by stefan-balke** [comment]
Yes, you are right. DOI not required. However, often those are the most interesting ideas :)

But low prio.

**Comment by alastair** [comment]
fyi, for 2019 we added lbd papers to archives.ismir.net, but don't have a json data file or upload to zenodo. Do we want to add a json file in the same format as the year data files?
